lib/index.js

```diff
@@ -66,7 +66,7 @@ async function _handleResponse(target, thisArg, args) {
     // a 204 will not include a content-type header
     const contentType = response.headers.get('content-type');
     if(contentType && contentType.includes('json')) {
-      response.data = await response.json();
+      Object.defineProperty(response, 'data', {value: await response.json()});
```

Comment:
Enlighten me: what is this change for?

Reply:
with the old changes when I updated ky and ky-universal, I was getting this error in the tests - TypeError: Cannot set property data of #<Body> which has only a getter. When I debugged I found out that response.data has been deprecated and is now a getter https://github.com/node-fetch/node-fetch/blob/main/src/body.js#L182-L184. So, that change is to reassign the getter.